packages/accounts-2fa/2fa-server.js

```diff
@@ -13,15 +14,15 @@ Accounts._is2faEnabledForUser = selector => {
 
   if (typeof selector === 'string') {
     if (!selector.includes('@')) {
-      selector = { username: selector };
+      selector = { $or: [{ _id: selector }, { username: selector }] };
     } else {
       selector = { email: selector };
     }
   }
 
   const user = Meteor.users.findOne(selector) || {};
   const { services: { twoFactorAuthentication } = {} } = user;
-  return (
+  return !!(
```

Comment:
Needed to ensure that boolean is returned. If twoFactorAuthentication was `{}` it would return `undefined` instead of `false`.